webrecorder/web-replay-gen
# Add basic styles

(https://github.com/webrecorder/web-replay-gen/issues/6)

- Add [tailwindcss](https://tailwindcss.com/)
- Reset CSS and polish
- Show customizable logo and title at top of page
- Allow customization of `<replay-web-page>` `embed` option

### Manual testing
1. Run `npm install`
2. Optional: update `site.logoSrc` and `replay.embed`
3. Run `npm run serve`. Verify site renders correct logo and replay option

### Screenshots
<img width="973" alt="Screen Shot 2022-08-23 at 8 59 25 PM" src="https://user-images.githubusercontent.com/4672952/186326733-fa9dafa1-8a45-4510-914c-7c62df657931.png">
